modules/docs/src/02-the-smithy-idl/02-traits.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_label: Smithy traits
+title: Smithy traits
+---
+
+Smithy comes with a powerful annotation system. Annotations in smithy are called [traits](https://awslabs.github.io/smithy/1.0/spec/core/model.html#applying-traits-to-shapes).
+
+These traits let you associate protocol-specific details to your data models and services.
+
+For instance, an operation can be labelled as compatible with http semantics using the [http traits](https://awslabs.github.io/smithy/1.0/spec/core/http-traits.html?highlight=http%20traits#http-binding-traits):
+
+```kotlin
```

Comment:
random question, what highlighter is docusaurus using? Any chance we could plug a custom grammar into it?

Reply:
It is using Prism, and yes you can add custom grammars to it afaik